backend.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+# main.py
```

Comment:
Leftover? Or is this just meant to signify that these files are in some sort of `main` module?